targets/csharp/UnittestRunner/UUnitTestRunner.cs

```diff
@@ -5,12 +5,11 @@
 using PlayFab.ClientModels;
 using System.Threading.Tasks;
 
+#pragma warning disable 0649, 0414
```

Comment:
If you can remove the _onCompleted, then we can remove the warning supressions (they are both for unused fields/ unused assignments)

Reply:
There's other unused in this file as well, so I left them.